main.tf

```diff
@@ -0,0 +1,95 @@
+resource "aws_elasticsearch_domain" "es" {
+  domain_name           = "${var.project}-${var.environment}-${var.name}"
+  # access_policies       = "${var.access_policies}"
```

Comment:
why is this commented?

Reply:
Ah damn, leftover. That shouldn't be there at all as it causes a continues change. It's better to attach the access policies separately like:

```
module "elasticsearch" {
  source                   = "github.com/skyscrapers/terraform-awselasticsearch?ref=initial"
  name                     = "es"
  project                  = "${var.project}"
  environment              = "${terraform.workspace}"
  version                  = "5.5"
  instance_count           = "${var.es_instance_count}"
  instance_type            = "${var.es_instance_type}"
  volume_size              = "${var.es_volume_size}"
  vpc_id                   = "${data.terraform_remote_state.static.vpc_id}"
  subnet_ids               = ["${slice(data.terraform_remote_state.static.db_subnets,0,var.es_instance_count)}"]
  dedicated_master_enabled = false
  access_policies          = "${data.aws_iam_policy_document.es_policy.json}"

  advanced_options {
    "indices.fielddata.cache.size"           = ""
    "rest.action.multi.allow_explicit_index" = "true"
  }
}

resource "aws_elasticsearch_domain_policy" "es_policy" {
  domain_name = "${module.elasticsearch.domain_name}"

  access_policies = "${data.aws_iam_policy_document.es_policy.json}"
}
```

I'll add it to the docs.